src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -29,6 +29,8 @@ import {
 } from "@madie/madie-util";
 import MultipleSelectDropDown from "./MultipleSelectDropDown";
 import MeasureGroupsWarningDialog from "./MeasureGroupWarningDialog";
+
+import { values } from "lodash";
```

Comment:
Looks like Lodash is already imported and also it looks like this is not used anywhere.